src/coreclr/jit/emitxarch.h

```diff
@@ -690,10 +697,23 @@ void emitIns_AX_R(instruction ins, emitAttr attr, regNumber ireg, regNumber reg,
 void emitIns_SIMD_R_R_I(instruction ins, emitAttr attr, regNumber targetReg, regNumber op1Reg, int ival);
 
 void emitIns_SIMD_R_R_A(instruction ins, emitAttr attr, regNumber targetReg, regNumber op1Reg, GenTreeIndir* indir);
```

Comment:
Do we not have any cases that can hit `emitIns_SIMD_R_R_A` for the broadcast support?

Reply:
I would expect at least some scenario to hit it, since `_A` is the general purpose addressing mode such as for `[base+index*scale]`

Reply:
The following case with Avx2 API call may hit this method:
```C#
public unsafe static Vector256<float> Foo(float f)
{
	Vector256<float> Vec = Avx2.BroadcastScalarToVector256(&f);
	Vector256<float> VecCns = Vector256.Create(1.0f,2.0f,3.0f,4.0f,5.0f,6.0f,7.0f,8.0f);
	return Vector256.Add(Vec, VecCns);
}
```

Do we also wanna cover this case?

Reply:
I believe so, yes. We would ideally be covering all 3 possible addressing modes.